docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+
+[Kubernetes on DCOS](http://mesosphere.com/kubernetes) adds Kubernetes to Mesosphere’s Datacenter Operating System (DCOS) in order to help enterprises unlock the next generation of datacenter scale, efficiency and automation. [Kubernetes](http://kubernetes.io/) and the Mesosphere [DCOS](http://mesosphere.com/learn/) together provide a powerful platform for building and deploying microservice-based applications and operate containerized (e.g., using Docker) applications in a scalable, elastic and secure manner. The Mesosphere DCOS, which is based on Apache Mesos, is integrating Kubernetes as a standard component which will be made available as part of the Mesosphere DCOS Community Edition (free) and the Mesosphere DCOS Enterprise Edition (paid).
+
+In this document, you'll learn how to use Kubernetes on the Mesosphere DCOS in order to build scalable, resilient and secure application architectures. You'll see how these principles apply to real-world scenarios through an example deployment.
+
+## 1. Terminology
+
+**Docker**, [docker.com](https://www.docker.com/): is a popular container solution that enables deployable apps to be quickly assembled from components and eliminates the friction between development, QA and production environments. It consists of 1. the Docker Engine, a portable, lightweight runtime and packaging tool, and 2. Docker Hub, a cloud service for sharing applications and automating workflows. For on-premise deployments, a private registry can be used to serve the Docker images.
+
+**Kubernetes**, [kubernetes.io](http://kubernetes.io/): is an open source orchestration system for Docker containers. It actively manages workloads to ensure that their state matches the users declared intentions. Kubernetes provides declarative primitives—such as pods, labels and services—to group the Docker containers that make up an application into logical units, for easy management and discovery.
+
+**Mesosphere DCOS**, [mesosphere.com](http://mesosphere.com/product/): provides a highly-portable, operations-driven environment that is fault-tolerant, scalable, elastic and supports multi-service deployments. It seamlessly supports both Docker (isolation-level) and Kubernetes (service-level) in an enterprise-grade, [multi-services](http://www.slideshare.net/timothysc/apache-coneu) environment.
+
+**Multi-service**. The ability of a platform to run multiple operational and analytical workloads on the same cluster, resulting in resource-efficiency, while providing elasticity and isolation capabilities. For example, Web servers (like nginx), App servers (such as node.js) using relational databases (e.g., MySQL) for the operational part can be used together with Hadoop/HDFS, Spark, Kafka, etc. for the analytics part, in one cluster.
+
+## 2. Use Cases
+
+Kubernetes on DCOS delivers the most value for those use cases that share one or more of the following characteristics:
+
+* Production systems employing very large clusters, in the 1000s of nodes.
```

Comment:
No problem with the overall point, but I'm a little wary of the "1000s of nodes" part because Kubernetes does not scale to 1000s of nodes yet. I don't think there's a simple way to explain here that the Mesos cluster can be 1000s of nodes but the Kubernetes portion can't (yet) be that large, so maybe just leave out the "1000s of nodes" part?